k9mail/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -69,53 +69,57 @@
 import timber.log.Timber;
 
 
-public class LocalFolder extends Folder<LocalMessage> implements Serializable {
-
-    private static final long serialVersionUID = -1973296520918624767L;
+public class LocalFolder extends Folder<LocalMessage> {
     private static final int MAX_BODY_SIZE_FOR_DATABASE = 16 * 1024;
-    static final long INVALID_MESSAGE_PART_ID = -1;
+    private static final long INVALID_MESSAGE_PART_ID = -1;
+
 
     private final LocalStore localStore;
     private final AttachmentInfoExtractor attachmentInfoExtractor;
 
-    private String mName = null;
-    private long mFolderId = -1;
-    private int mVisibleLimit = -1;
+
+    private String folderName = null;
+    private long folderId = -1;
```

Comment:
Same as above. I'd make this simply `id`. This is also in line with my suggestion for `LocalMessage`.